packages/core/src/App/app.jsx

```diff
@@ -35,6 +43,24 @@ const App = ({ root_store }) => {
         setUrlLanguage(getLanguage());
     }, []);
 
+    React.useEffect(() =>
+        autorun(async () => {
```

Comment:
is this needed here?

Reply:
Yes as we need to listen the changes to `root_store` and avoid empty values

Reply:
@salar-fs I think putting an `autorun` in the App component is very inefficient for something we only need to be react to on init. If you agree, please move this entire logic and view to `<AppContents />`, and instead of using `autorun`, you can just `connect` to the value you want to listen to.

Reply:
@mahdipourziaei-binary I do agree.
Moved all the stuff to `<AppContents />`